taskoutput/task_log.go

```diff
@@ -80,29 +71,27 @@ func (o TaskLogs) Get(ctx context.Context, env evergreen.Environment, taskOpts T
 	})
 }
 
-func (o TaskLogs) getLogName(taskOpts TaskOptions, logType TaskLogType) string {
+func (o TaskLogOutput) getLogName(taskOpts TaskOptions, logType TaskLogType) string {
 	prefix := fmt.Sprintf("%s/%s/%d/%s", taskOpts.ProjectID, taskOpts.TaskID, taskOpts.Execution, o.ID())
 
-	var filename string
+	var logTypePrefix string
 	switch logType {
 	case TaskLogTypeAgent:
-		filename = "agent"
+		logTypePrefix = "agent"
 	case TaskLogTypeSystem:
-		filename = "system"
+		logTypePrefix = "system"
 	case TaskLogTypeTask:
-		filename = "task"
+		logTypePrefix = "task"
 	default:
 		return prefix
```

Comment:
For my own understanding, is "all" logs treated the same as an empty/unset `TaskLogType`? Or will `TaskLogType` always be set/always be validated before `getLogName` is called?

Reply:
It should always be set and validated before `getLogName` is called!